packages/myst-cli-utils/src/index.ts

```diff
@@ -10,7 +10,6 @@ export {
 export { exec, makeExecutable } from './exec.js';
 export { clirun, tic } from './utils.js';
 export { isUrl } from './isUrl.js';
-export { Session, getSession } from './session.js';
```

Comment:
This super minimal session implementation isn't used anywhere (besides a single test file - easily replaced). I just removed it so we did not need to implement `fetch` in this package.